wlhosted/templates/legal/documents/privacy.html

```diff
@@ -12,12 +12,12 @@ <h2>{% blocktrans %}Personal Data Controller{% endblocktrans %}</h2>
 {% blocktrans with provider="Ing. Michal Čihař, Zdiměřická 1439/8, 149 00 Praha, CZ" reg_no="04705904" %}{{ provider }}, Reg. No. {{ reg_no }}{% endblocktrans %}
 </p>
 
-<p>{% blocktrans with privacy_contact="privacy@weblate.org" %}We have appointed a data protection officer who may be reached via {{privacy_contact}}.{% endblocktrans %}</p>
+<p>{% blocktrans with privacy_contact="privacy@weblate.org" %}Weblate has appointed a data protection officer who may be reached via {{privacy_contact}}.{% endblocktrans %}</p>
 
 <h2>{% blocktrans %}Personal Data we process{% endblocktrans %}</h2>
 
 <p>
-{% trans "We process only Personal Data User provides us by using our service:" %}
+{% trans "Weblate only process Personal Data the User provides us by using our service:" %}
```

Comment:
- [ ] Sort out clarifications of whether EU-GDPR sections are the same for all.

Though this is modeled on the EU-GDPR, which means citizens in the EU eco zone, it applies to all?